layers/+source-control/git/packages.el

```diff
@@ -137,6 +137,8 @@
     :defer (spacemacs/defer)
     :init
     (progn
+      (push "magit: .*" spacemacs-useless-buffers-regexp)
+      (push "magit-.*"  spacemacs-useless-buffers-regexp)
```

Comment:
This ends up filtering valid buffer names like `magit-tutorial.org`. Please add the entire `magit-.*?: .*` regex.

I wish magit would add `*` at the beginning of their buffers so that they would be easyer to filter.

Reply:
Good call, updated 👍:sparkles: